### ISSUE TITLE
increase epochs number when the learning rate decreases

### ISSUE THREAD

**Issue by khawkhaa** [rstudio/keras#646]
Hi,

I apologize if this has already been asked, but I have not find in the Keras R documentation the answer to my question. I am using :

`callback_reduce_lr_on_plateau(monitor = "val_loss", factor = 0.1, patience=50, verbose=1, mode=c('min'))`

How can I increase the epoch’s number when I decrease the learning rate value. 

I am looking to have patience=50 when the lr = 0.2 at the beginning, and a highest number of epochs as the learning rate decreases (for example patience =100 when lr=0.02, patience =1000 when lr=0.002) to give more time to the algorithm when the lr is small.

**Comment by skeydan** [comment]
I think you would have to write your own callback to implement something like this.
